### PR TITLE
updating Z versions per support

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

File: modules/installation-requirements-user-infra.adoc [hunk 1]
```diff
@@ -115,7 +115,7 @@ ifdef::ibm-z[]
 
 You can install {product-title} version {product-version} on the following IBM hardware:
 
-* IBM Z, version z13 and later and z13s and later
+* IBM Z, version 13, 14, or 15
```
Comment (sferich888) [inline modules/installation-requirements-user-infra.adoc]: nit: version should be plural

Comment (kalexand-rh) [reply modules/installation-requirements-user-infra.adoc]: Thank you! I have fixed that nit.

**Pull request merged**